src/Ads/AdsRecommendationsService.php

```diff
@@ -156,6 +205,45 @@ public function get_google_recommendations( $args ): array {
 				$campaign = $row->getCampaign();
 				$customer = $row->getCustomer();
 
+				$recommendation_type    = RecommendationType::name( $recommendation->getType() );
+				$recommendation_details = [];
+
+				// Add recommendation details depending on the type.
+				if ( in_array( $recommendation_type, [ 'CAMPAIGN_BUDGET', 'MARGINAL_ROI_CAMPAIGN_BUDGET' ], true ) ) {
+					$budget_recommendation = $recommendation->getCampaignBudgetRecommendation();
+					$budget_options        = [];
+
+					foreach ( $budget_recommendation->getBudgetOptions() as $option ) {
+						$impact    = $option->getImpact();
+						$base      = $impact->getBaseMetrics();
+						$potential = $impact->getPotentialMetrics();
+
+						$budget_options[] = [
+							'budget_amount_micros' => $option->getBudgetAmountMicros(),
```

Comment:
Do we needs to use `$this->from_micro` similar to https://github.com/woocommerce/google-listings-and-ads/blob/bbb59d6aab89a85d2c6d38965b27a3692b566cd7/src/API/Google/BudgetRecommendations.php#L129